Documentation/admin_guide.md

```diff
@@ -209,3 +209,11 @@ If the request times out, it indicates two possibilities:
 2. the majority of the cluster is not functioning.
 
 If timeout happens several times continuously, administrators should check status of cluster and resolve it as soon as possible.
+
+### Deployment
+
+#### CPU cores
+
+In default, etcd process uses only one CPU core.
+
+Set environment variable `GOMAXPROCS=$number_of_cpu_cores` when starting etcd to allow etcd process use desired number of CPU cores. etcd consumes computation resources heavily, so using multiple CPU cores is critical for good performance.
```

Comment:
If you plan to use etcd under heavy load, you can give more cores to etcd to reduce latency. To use more cores, set ...